internal/build.go

```diff
@@ -159,6 +159,38 @@ func assertRuleIdsWithoutSnykPrefix(rules []util.Rule) error {
 	return nil
 }
 
+func assertValidSeverityLevel(rules []util.Rule) error {
+	invalidRulesPaths := []string{}
+
+	for _, rule := range rules {
+		var isValidSeverityLevel = func(severityLevel string) bool {
+			for _, validSeverityLevel := range ValidSeverityLevels {
+				if validSeverityLevel == severityLevel {
+					return true
+				}
+			}
+			return false
+		}
+
+		if !isValidSeverityLevel(rule.SeverityLevel) {
+			invalidRulesPaths = append(invalidRulesPaths, rule.Path)
+		}
+	}
+
+	if len(invalidRulesPaths) > 0 {
+		errMessage := "We cannot create a bundle for your custom rules." +
```

Comment:
thought: Have we reached out to the content team to check with them if there are any modifications we should make to this user message? 

Do you think we should?

Reply:
We reached out for the rest of these messages and I used what they recommended for the CLI validation [here](https://snyk.slack.com/archives/CM5LMKKQR/p1653404088692759) for these messages. So I don't think we need to reach out again 😄